src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -49,6 +50,11 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         try {
             $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, false);
         } catch (\Exception $e) {
+        } catch (\Throwable $e) {
+            $e = new FatalThrowableError($e);
```

Comment:
the class used by ``ReflectionProperty`` may need to be altered below to account for Error though (but it is the case anyway as the previous may also be an Error)